modular_splurt/code/modules/mob/living/emotes.dm

```diff
@@ -1627,3 +1627,12 @@ To add randomization to your emote, copy and paste this line of code:
 	emote_cooldown = 4 SECONDS
 	vary = FALSE
 	volume = 30
+
+/datum/emote/sound/human/squirm
+	key = "squirm"
+	key_third_person = "squirm"
+	name = "squirm"
+	message = "squirms in place!"
+	message_mime = "squirms in place!"
+	emote_type = EMOTE_VISIBLE
+	emote_cooldown = 0.8 SECONDS // mrrp mrrp meow
```

Comment:
```suggestion
	emote_cooldown = 0.8 SECONDS
```

Reply:
Pfft! pretty sure that came from the *mrrp emote and I never took it off lol